src/rust/ide/src/view/node_editor.rs

```diff
@@ -511,8 +526,12 @@ impl GraphEditorIntegratedWithControllerModel {
     }
 
     /// Handle notification received from controller about values having been entered.
-    pub fn on_node_entered(&self, _id:double_representation::node::Id) -> FallibleResult<()> {
+    pub fn on_node_entered(&self, local_call:&LocalCall) -> FallibleResult<()> {
+        let definition = graph_editor::MethodPointer(Rc::new(local_call.definition.clone()));
```

Comment:
I'd add a `new` function for MethodPointer, or even consider implementing From. 
If this is wrapper that wants to abstract away Rc<thing>, then it shouldn't force users to explicitly invoke Rc::new.